### DISCUSSION TITLE
Add PlayerAFKToggleEvent

### DISCUSSION THREAD

**Discussion by flashbtw** [EssentialsX/Essentials#4542] [Ideas and Feature Suggestions] [answered]
Make a event that triggers, whenever a player goes Auto-AFK or types /afk
You can extend it by making a method to get the status. So if the player is afk or not. 

Have a nice day :)

**Comment by JRoy** [comment, marked as answer]
Essentials already has an event for this: [AfkStatusChangeEvent](https://jd-v2.essentialsx.net/net/ess3/api/events/afkstatuschangeevent). You can use `getCause()` to get the cause for the change and `getValue()` to get the new afk status (true would mean the player is now afk, otherwise false).

**Reply by flashbtw** [reply]
Oh, sorry, I didn't see that. Anyway thanks for the answer and I'm going to build it into my plugin :D
